examples/mariadb-claim.yaml

```diff
@@ -1,11 +1,8 @@
 apiVersion: azure.platform.upbound.io/v1alpha1
-kind: SQLInstance
+kind: XSQLInstance
```

Comment:
So it is not claim anymore but XR, so does not match the filename promise

Reply:
renamed to `mariadb-xr.yaml` as composition-tests don't support claims